Editors/vscode/package.json

```diff
@@ -62,6 +62,20 @@
                     "description": "Traces the communication between VS Code and the SourceKit-LSP language server."
                 }
             }
-        }
+        },
+        "taskDefinitions": [
+            {
+				"type": "swift-package",
```

Comment:
Could you change this to use spaces instead of tabs to match the rest of the document?